src/romaine/parser/__init__.py

```diff
@@ -0,0 +1,34 @@
+from romaine.parser import (
+    simple,
+    multiline,
+    step,
+    section,
+    feature,
+    )
+from romaine.parser.exceptions import (  # noqa
+    UnclosedPythonishString,
+    MalformedTableError,
+    )
+
+
+class Parser(object):
+    """
+        Gherkin parser for romaine core.
```

Comment:
Ergh- that style always looks horrible to me. I much prefer doing the docstrings with the same indentation approach used everywhere else (given that I newline it immediately, because it looks ugly to have text immediately following the triple quotes).

How attached are you to this element of style? I can go through and change it, but currently that style is used everywhere (including in the core).